Misc/NEWS.d/next/Library/2018-09-28-22-18-03.bpo-34828.5Zyi_S.rst

```diff
@@ -0,0 +1,3 @@
+Sqlite3.iterdump now correctly deletes the "sqlite_sequence" table and
+inserts the rows from the dumped db. Just like the command ".dump" in
+sqlite3 does.
```

Comment:
I'd shorten this to something more accessible. For example:

    :meth:`~sqlite3.Connection.iterdump` now handles databases that use ``AUTOINCREMENT`` in one or more tables.

See the [dev guide](https://devguide.python.org/documenting/#additional-markup-constructs) for how to add reST markups.

Reply:
You can also add _"Patch by Joel Klimont"_ at the end of the NEWS entry.